src/mpid/ch3/channels/sock/src/sock.c

```diff
@@ -2183,13 +2183,16 @@ int MPIDI_CH3I_Sock_post_close(struct MPIDI_CH3I_Sock *sock)
     if (pollinfo->type == MPIDI_CH3I_SOCKI_TYPE_COMMUNICATION) {
         if (MPIDI_CH3I_SOCKI_POLLFD_OP_ISSET(pollfd, pollinfo, POLLIN | POLLOUT)) {
             /* --BEGIN ERROR HANDLING-- */
+            /* comment out unused code. FIXME: understand original intention, then delete. */
+            /*
             int event_mpi_errno;
 
             event_mpi_errno =
                 MPIR_Err_create_code(MPI_SUCCESS, MPIR_ERR_RECOVERABLE, __func__, __LINE__,
                                      MPIDI_CH3I_SOCK_ERR_SOCK_CLOSED, "**sock|close_cancel",
                                      "**sock|close_cancel %d %d", pollinfo->sock_set->id,
                                      pollinfo->sock_id);
+            */
```

Comment:
Agreed. My intention was to get some review attention in case I miss something. If I delete the code right away, then I don't have a place add review notes (the FIXME comment). 

I'll delete this.

Reply:
Has this been deleted?

Reply:
FYI, deleted code shows up in the patch for review too.  So we should really try to avoid creating dead code.